src/haxe/net/impl/WebSocketJs.hx

```diff
@@ -31,6 +31,14 @@ class WebSocketJs extends WebSocket {
                 //haxe.io.Int8Array
                 //js.html.ArrayBuffer
                 trace('Unhandled websocket onmessage ' + m);
+            } else if (Std.is(m, js.html.Blob)) {

```

Comment:
Is it neccessary as the whole class is already only imported in an #if js in haxe.net.WebSocket.create()?